pkg/julia/manifest/parse_test.go

```diff
@@ -0,0 +1,98 @@
+package julia
+
+import (
+	"os"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func TestParse(t *testing.T) {
+	tests := []struct {
+		name     string
+		file     string // Test input file
+		want     []types.Library
+		wantDeps []types.Dependency
+	}{
+		{
+			name:     "Manifest v1.6",
+			file:     "testdata/primary/Manifest_v1.6.toml",
```

Comment:
You only use `Manifest.toml` files. 
Remove other files ( like `Project.toml`, `README.md`, etc...) files, please.

Reply:
Removed